__tests__/Port.test.js

```diff
@@ -0,0 +1,44 @@
+const Port = require("../src/Port");
+
+describe('Port', () => {
+    describe('have names and can add and remove ships', () => {
+        let port;
+        let ship;
+        let royalCaribbean;
+        let queenMary;
+
+        beforeEach(() => {
+             port = new Port('Rotterdam');
+             let ship = jest.fn();
```

Comment:
You dont need to instantiate the variable here as youve already done it on line 6:
```
             ship = jest.fn();
```